test/directed-requester.js

```diff
@@ -0,0 +1,91 @@
+import test from 'ava';
+import r from 'randomstring';
+import LogSuppress from '../lib/log-suppress';
+
+LogSuppress.init(console);
+
+const environment = r.generate();
+const { DirectedRequester, Responder } = require('../')({ environment });
+
+test.cb('Supports directed requests using callbacks', (t) => {
```

Comment:
Please have only two responders in this test. Send two different messages with the same requester, see that both ends up with the first responder and not the second (as it would be the case with the actual flow).

Otherwise this only tests that a directed requester can send and receive requests, not necessarily directed.